### PR TITLE
Concept component

### PR DESCRIPTION
**Pull request by machang19**
I made concept a separate component. I also switched the image carousel to an image swiper so it has the dots underneath and you only see one image at a time.

### CODE REVIEW

File: peerdea-native/components/Concept.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+// PICK UP HERE
+//TODO: change infrastructure of this file to make state hold multple values of what a concet is
+
+export default class Concept extends React.Component {
+
+constructor(props) {
+    super(props);
+    this.state = {
+      concept: props.concept,
```
Comment (ykotturi) [inline peerdea-native/components/Concept.js]: At this point, is props.concept empty?

Comment (machang19) [reply peerdea-native/components/Concept.js]: No it is not empty it is given as arguments in the constructor when you create a concept. See line 68 in giveFeedbackScreen

**Comment by machang19** [comment]
> Works locally except for the image picker dots on ShareConceptScreen: they exist, they just don't update while scrolling. Dots updates on GiveFeedbackScreen.

Should be resolved now

**Comment by ykotturi** [comment]
The reason for this issue: "It was actually a weird bug with the Slider object itself that requires a workaround to only render a slider if it has more than one image in it"

**Pull request merged**